src/client/contextChannels.ts

```diff
@@ -21,10 +21,32 @@ import {getEventRouter} from './EventRouter';
  */
 export type ChannelId = string;
 
+/**
+ * A system channel will be global enough to have a presence across many apps. This gives us some hints
+ * to render them in a standard way. It is assumed it may have other properties too, but if it has these,
+ * this is their meaning.
+ */
```

Comment:
Would rewrite to start with a one-sentence description of what this is. "Defines the visual appearance of a system channel when presented in an app, for example, as part of a channel selector"? The "if' in the last sentence also feels odd, since these aren't optional properties

Reply:
I copied all the doc comments on these from the FDC3 spec docs. They could probably do with some general improvements.

Reply:
Also, the "if" is because they are option in the spec. I'm rewriting this whole comment to make more sense